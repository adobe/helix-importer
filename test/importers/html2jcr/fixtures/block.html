
<!DOCTYPE html>
<html>
  <head>
    <title>Sites Franklin Example</title>
    <link rel="canonical" href="https://main--xwalk-example--buuhuu.hlx.page/">
    <meta name="description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
    <meta property="og:title" content="Sites Franklin Example">
    <meta property="og:description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
    <meta property="og:url" content="https://main--xwalk-example--buuhuu.hlx.page/">
    <meta property="og:image" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
    <meta property="og:image:secure_url" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:title" content="Sites Franklin Example">
    <meta name="twitter:description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
    <meta name="twitter:image" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
    <meta name="foobar" content="hello world">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src="/scripts/lib-franklin.js" type="module"></script>
    <script src="/scripts/scripts.js" type="module"></script>
    <link rel="stylesheet" href="/styles/styles.css">
  </head>
  <body>
    <header></header>
    <main>
      <div>
        <div class="simple-block">
          <div>
            <div>
              This is a simple block.
            </div>
          </div>
        </div>
        <div class="cards alpha">
          <div>
              <div>
                  <img src="/content/dam/test12/unmatched-speed.png" />
              </div>
              <div>
                  <p><strong>Content at scale</strong></p>
                  <p>AEM allows you to publish more content in shorter time with smaller teams</p>
              </div>
          </div>
          <div>
              <div>
                  <img src="/content/dam/test12/content-at-scale.png" />
              </div>
              <div>
                  <p><strong>Unmatched speed</strong></p>
                  <p>AEM is the fastest way to publish, create, and serve websites</p>
              </div>
          </div>
          <div >
              <div>
                  <img src="/content/dam/test12/uncertainty-eliminated.png" />
              </div>
              <div>
                  <p><strong>Uncertainty eliminated</strong></p>
                  <p>Preview content at 100% fidelity, get predictable content velocity, and shorten project durations</p>
              </div>
          </div>
          <div>
              <div>
                  <img src="/content/dam/test12/widen-the-talent-pool.png" />
              </div>
              <div>
                  <p><strong>Widen the talent pool</strong></p>
                  <p>Authors on AEM use Microsoft Word, Excel or Google Docs and need no training</p>
              </div>
          </div>
          <div>
              <div>
                  <img src="/content/dam/test12/low-code-way-to-developer-productivity.png" />
              </div>
              <div>
                  <p><strong>The low-code way to developer productivity</strong></p>
                  <p>Say goodbye to complex APIs spanning multiple languages. Anyone with a little bit of HTML, CSS, and JS can build a site on AEM.</p>
              </div>
          </div>
          <div>
              <div>
                  <img src="/content/dam/test12/headless-is-here.png" />
              </div>
              <div>
                  <p><strong>Headless is here</strong></p>
                  <p>Go directly from Microsoft Excel or Google Sheets to the web in mere seconds. Sanitize and collect form data at extreme scale with AEM Forms.</p>
              </div>
          </div>
          <div>
              <div>
                  <img src="/content/dam/test12/peak-performance.png" />
              </div>
              <div>
                  <p><strong>Peak performance</strong></p>
                  <p>Use AEM&#39;s serverless architecture to meet any traffic need. Use AEM&#39;s PageSpeed Insights Github action to evaluate every Pull-Request for Lighthouse Score.</p>
              </div>
          </div>
        </div>
      </div>
    </main>
    <footer></footer>
  </body>
</html>
