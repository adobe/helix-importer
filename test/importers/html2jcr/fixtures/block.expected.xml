<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" jcr:title="Sites Franklin Example" sling:resourceType="core/franklin/components/page/v1/page">
        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
            <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
                <block sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="Simple Block" text="This is a simple block." model="simple-block"/>
                <block_0 sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="Cards" filter="card" classes="[alpha]" model="cards">
                    <item jcr:primaryType="nt:unstructured" name="Card" image="/content/dam/test12/unmatched-speed.png" text="&lt;p>&lt;strong>Content at scale&lt;/strong>&lt;/p>&lt;p>AEM allows you to publish more content in shorter time with smaller teams&lt;/p>" model="card"/>
                    <item_0 jcr:primaryType="nt:unstructured" name="Card" image="/content/dam/test12/content-at-scale.png" text="&lt;p>&lt;strong>Unmatched speed&lt;/strong>&lt;/p>&lt;p>AEM is the fastest way to publish, create, and serve websites&lt;/p>" model="card"/>
                    <item_1 jcr:primaryType="nt:unstructured" name="Card" image="/content/dam/test12/uncertainty-eliminated.png" text="&lt;p>&lt;strong>Uncertainty eliminated&lt;/strong>&lt;/p>&lt;p>Preview content at 100% fidelity, get predictable content velocity, and shorten project durations&lt;/p>" model="card"/>
                    <item_2 jcr:primaryType="nt:unstructured" name="Card" image="/content/dam/test12/widen-the-talent-pool.png" text="&lt;p>&lt;strong>Widen the talent pool&lt;/strong>&lt;/p>&lt;p>Authors on AEM use Microsoft Word, Excel or Google Docs and need no training&lt;/p>" model="card"/>
                    <item_3 jcr:primaryType="nt:unstructured" name="Card" image="/content/dam/test12/low-code-way-to-developer-productivity.png" text="&lt;p>&lt;strong>The low-code way to developer productivity&lt;/strong>&lt;/p>&lt;p>Say goodbye to complex APIs spanning multiple languages. Anyone with a little bit of HTML, CSS, and JS can build a site on AEM.&lt;/p>" model="card"/>
                    <item_4 jcr:primaryType="nt:unstructured" name="Card" image="/content/dam/test12/headless-is-here.png" text="&lt;p>&lt;strong>Headless is here&lt;/strong>&lt;/p>&lt;p>Go directly from Microsoft Excel or Google Sheets to the web in mere seconds. Sanitize and collect form data at extreme scale with AEM Forms.&lt;/p>" model="card"/>
                    <item_5 jcr:primaryType="nt:unstructured" name="Card" image="/content/dam/test12/peak-performance.png" text="&lt;p>&lt;strong>Peak performance&lt;/strong>&lt;/p>&lt;p>Use AEM's serverless architecture to meet any traffic need. Use AEM's PageSpeed Insights Github action to evaluate every Pull-Request for Lighthouse Score.&lt;/p>" model="card"/>
                </block_0>
            </section>
        </root>
    </jcr:content>
</jcr:root>